src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -229,6 +228,7 @@ export class MatTabLink extends _MatTabLinkMixinBase
     // Manually create a ripple instance that uses the tab link element as trigger element.
     // Notice that the lifecycle hooks for the ripple config won't be called anymore.
     this._tabLinkRipple = new MatRipple(_elementRef, ngZone, platform, globalOptions);
+    this._tabLinkRipple.ngOnInit();
```

Comment:
It feels weird having to call the Angular lifecycle hook manually. Shouldn't the renderer support attaching to arbitrary elements?

Reply:
Addressed the solution we discussed on Slack. I still kind of feel that the first commit is more clean and it's very uncommon that we want to use the `RippleRenderer` without the `MatRipple` directive.